src/Bicep.Core/Semantics/FileSymbol.cs

```diff
@@ -31,6 +31,7 @@ public class FileSymbol : Symbol, ILanguageScope
 
             // TODO: Avoid looping 6 times?
             this.ImportDeclarations = declarations.OfType<ImportedNamespaceSymbol>().ToImmutableArray();
+            this.MetadataDeclarations = declarations.OfType<MetadataSymbol>().ToImmutableArray();
```

Comment:
I think we'll want to avoid the symbol uniqueness check with the `meta` keyword - please could you add a test for the following and ensure it is permitted:

```bicep
param foo string
meta foo = 'sdf'
```